isaac2/Assets/Scripts/EnemyScripts/EnemyHealth.cs

```diff
@@ -1,191 +1,54 @@
 using System;
 using System.Collections;
 using System.Collections.Generic;
+//using System.Numerics;
 using Unity.VisualScripting;
 using UnityEngine;
 
-public class EnemyHealth : MonoBehaviour
+public class EnemyHealth : Health
 {
     /// <summary>
-    /// Health class stores entity health and deals with setting, incrementing,
-    /// and fetching both current and max health.
-    /// Also detects death and overheal.
-    /// In theory this was meant to be shared between Player and Enemy,
-    /// but I am not sure if that is a good idea or not.
-    /// </summary>
-
-    // TODO: If we want health bars they could be handled here
-
-    [SerializeField] private int maxHealth;
-    [SerializeField] private int curHealth;
-    [SerializeField] private bool isDead;
-    
-    // Start is called before the first frame update
-    void Start()
-    {
-        curHealth = maxHealth;
-    }
+    ///     Enemy implentation of Health abstract class, 
+    ///     which handles Entity max & cur health, damage,
+    ///     healing, death, etc.
+    ///     Implements Death() method, which handles Enemy death.
+    ///     Also handles corpse spawning, (future) death animation,
+    ///     and (future) loot dropping.
+    /// </summary> 
 
-    // Update is called once per frame
-    //void Update()
-    //{
-        // Could implement gradual natural healing over time I guess?
-        // otherwise can we delete this method entirely (Unity question)
-    //}
+    [SerializeField] private GameObject resourcePrefab;
```

Comment:
Do we still need this if we have a loot table in EnemyInventory?

Reply:
Good catch forgot to remove that